pkgs/development/libraries/libidn/default.nix

```diff
@@ -1,24 +1,22 @@
 { fetchurl, lib, stdenv, libiconv }:
 
 stdenv.mkDerivation rec {
-  name = "libidn-1.36";
+  pname = "libidn";
+  version = "1.37";
 
   src = fetchurl {
-    url = "mirror://gnu/libidn/${name}.tar.gz";
-    sha256 = "0f20n634whpmdwr81c2r0vxxjwchgkvhsr1i8s2bm0ad6h473dhl";
+    url = "mirror://gnu/libidn/${pname}-${version}.tar.gz";
+    sha256 = "1hljls9rkilh04dwiwg388wk8djkac8idijrdysqq2nqbh6rg3iw";
   };
 
   outputs = [ "bin" "dev" "out" "info" "devdoc" ];
 
-  # broken with gcc-7
-  #doCheck = !stdenv.isDarwin && !stdenv.hostPlatform.isMusl;
+  doCheck = true;
```

Comment:
Isn't the default true? Or was that pythonPackages?

Reply:
It looks like the default is false (`pkgs.config.doCheckByDefault`).

Anyway, I’ll change shortly to use the default behavior and remove this line. Then, having tests is just a matter of tessing `doCheckByDefault`.

Reply:
This is changed.